testsuite/src/it/java/io/quarkus/ts/startstop/ArtifactGeneratorTest.java

```diff
@@ -104,7 +104,7 @@ public class ArtifactGeneratorTest {
             "micrometer-registry-prometheus",
             "quarkus-openshift-client",
             "quarkus-smallrye-graphql-client",
-            "qpid-jms",
+//            "qpid-jms",  // not part of the initial RHBQ 2.13 release
```

Comment:
Why not remove this line for good? Or we expected it to be added in later build?

Reply:
exactly